src/main/java/me/ferdz/placeableitems/Config.java

```diff
@@ -1,21 +1,50 @@
 package me.ferdz.placeableitems;
 
-import java.util.List;
-import java.util.Set;
-import java.util.stream.Collectors;
-
-import net.minecraft.core.registries.BuiltInRegistries;
-import net.minecraft.resources.ResourceLocation;
-import net.minecraft.world.item.Item;
-import net.neoforged.bus.api.SubscribeEvent;
-import net.neoforged.fml.common.EventBusSubscriber;
-import net.neoforged.fml.event.config.ModConfigEvent;
+import me.ferdz.placeableitems.init.PlaceableItemsBlockRegistry;
 import net.neoforged.neoforge.common.ModConfigSpec;
 
+import java.util.HashMap;
+import java.util.Map;
+
 // An example config class. This is not required, but it's a good idea to have one to keep your config organized.
 // Demonstrates how to use Neo's config APIs
 public class Config {
     private static final ModConfigSpec.Builder BUILDER = new ModConfigSpec.Builder();
 
+    public static final PlaceableItemConfigs PLACEABLE_ITEMS = new PlaceableItemConfigs(BUILDER);
+
     static final ModConfigSpec SPEC = BUILDER.build();
-}
+
+    public static class PlaceableItemConfigs {
+
+        private final Map<String, ModConfigSpec.BooleanValue> itemToggles = new HashMap<>();
+
+        public PlaceableItemConfigs(ModConfigSpec.Builder builder) {
+            builder.push("items_toggle");
+
+            for (String itemId : PlaceableItemsBlockRegistry.ALL_PLACEABLE_ITEM_IDS) {
+                String configKey = "enable_placeable_" + itemId.replace("_", "");
+
+                ModConfigSpec.BooleanValue configValue = builder.define(configKey, true);
```

Comment:
Please make use of placeholder strings instead of multiple individual strings, and do not strip the `_` from the ID. For example, here you can do something like this:
```
String configKey = "enable_placeable_" + itemId;
String itemName = Component.translatable(item.getDescriptionId()).toString();
String translation = Component.translatable("placeableitems.configuration.enable_placeable", itemName).toString();
ModConfigSpec.BooleanValue configValue = builder.translation(translation).define(configKey, true);
```
Then only create one entry in the lang json with a placeholder string